### ISSUE TITLE
Even when downloading and installing the usersetup windows will only open whenrun by a local admin.

### ISSUE THREAD

**Issue by shoitinga** [microsoft/vscode#160171]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:
- OS Version:

Steps to Reproduce:

1.
2.

**Comment by gjsjohnmurray** [comment]
Did you accept the default location when you ran the installer?

Did you use "Run as administrator" when you ran the installer? I hope you didn't.

**Comment by shoitinga** [comment]
Working on windows 11. First I downloaded the user version, installed, accepted default location.
Upon starting I only noticed a vague mirror window. Then a message appeared. Window not responding. 
I uninstalled and tried the administrator version, same problem. Only IT support was able to get it running, when they opened as domain administrator. Uninstalled it.
Tried the user version again, same problem.
